features/web-audio.yml

```diff
@@ -0,0 +1,190 @@
+name: Web Audio
```

Comment:
Should the name be "Web Audio" with Audio capitalized? "Web audio" looks wrong to me because it's not about audio generically, but this specific API. "Web Audio API" is the spec name.

Reply:
Web Audio feels much better. That's how everybody talks about it.